validator/specs/lib/validateTilesetSpec.js

```diff
@@ -77,3 +78,98 @@ describe('validateTileset', function() {
             }), done).toResolve();
     });
 });
+
+
+describe('validateBoundingVolume', function() {
+   var regionTileset = {
+       "root": {
+           "boundingVolume": {
+               "region": [10, 10, 10, 10, 10, 88]
+           },
+           "content": {
+               "boundingVolume": {
+                   "region":  [0, 0, 0, 0, 20, 30]
+               },
+           }
+       }
+   };
+
+    it('validates region inside region', function(done) {
+        expect(validateBoundingVolume(regionTileset)
+            .then(function(response) {
+                expect(response.result).toBe(true);
+                expect(response.message).toBe('Tileset is valid');
+            }), done).toResolve();
+    });
+
+    it('validates an invalid region inside region', function(done) {
```

Comment:
Slightly awkward wording. Maybe `Invalidates when a content region exceeds the tile region`.